Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+so it does not interfere with the projects timeline.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Fortunately, most of the materials for the test stand have been purchased. This eliminates much of the system cost. The most costly part of the project
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary, worst-case, scenario was determined by finding most of the parts
+on McMaster Carr. The total cost was approximately \$450. That would be the last part of the system needed before testing can occur. Accounting for unforseen cost then rounding upwards,
+the consquentical estimated cost is put at \$600 for the entirty of the project. A bills of materials is displayed in Appendix B.
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Very limited faculty support will be necessary. The SPEX propulsion team has taken care of most of the parts that will need the assistance from faculty.
+This included the flow system and how it operates. Is it safe and is it necessary? Those questions were answered by Dr. Garrick of the MET department.
+There may be some assistance required if the nozzle performance if far below nominal, but guessing from past work it will not be a problem. There may also
+be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+For the data analysis, if assistance is required for fully understanding it, Dr. Olles would be a good professor to ask. He has a PhD in rocket science
+so he could certainly help. Also, a SPEX member, Varun, has experience in flow simulation so he could probably understand the data sufficiently too.
+\subsection{Long-Term Vision}
+\label{sec:vision}
+
+The long term vision of this project is to be a building block to the next phase of space propulsion research. CREST, a mission that was theorized in the
+spring of 2016 by James Parkus, relies on a space propulsion system. It is the reason the SPEX propulsion subsystem was formed. Thus, it is natural for the
+CGT to assist in the furthering of this project as the complication of the propulsion system has been the main concern. The problem found with this was the required velocity change.
+The CubeSat would need to change its orbital velocity by 122 $ms^{-1}$, opposite its direction of motion. To do this with a cold gas thruster one needs a certain amount
+of propellant varing with the chamber pressure. If you increase the pressure, one can hold more propellant but the optimal value of both was far higher than we anticipated.
```

Comment:
Too much detail in the follow sections. Be concise when outlining future projects and applications.